### PR TITLE
Use GitHub Actions instead of Travis CI

### PR DESCRIPTION
**Pull request by sh0hei**
cf.
- https://xuwei-k.hatenablog.com/entry/2020/11/04/161400
- https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      fail-fast: false
+      matrix:
+        java: [8, 11]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.${{ matrix.java }}"
+      - uses: coursier/cache-action@v5
+      - run: sbt -v $(if [[ "${{matrix.sbt_version}}" != "" ]] ; then echo "^^${{matrix.sbt_version}}" ; fi) test checkScalariform
```
Comment (xuwei-k) [inline .github/workflows/ci.yml:19]:
```suggestion
      - run: sbt -v "+ test" checkScalariform
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11]
+    steps:
+      - uses: actions/checkout@v1
```
Comment (ocadaruma) [inline .github/workflows/ci.yml:14]: Let's use `checkout@v2.3.4` ? (latest)

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      - uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.${{ matrix.java }}"
+      - uses: coursier/cache-action@v5
+      - run: sbt -v "+ test" checkScalariform
+      - run: rm -rf "$HOME/.ivy2/local"
```
Comment (ocadaruma) [inline .github/workflows/ci.yml:20]: Could you elaborate why is this step necessary?

Comment (sh0hei) [reply .github/workflows/ci.yml]: sorry, this wasn't necessary 🙏 
It has been deleted 👍

**Review by ocadaruma — APPROVED** [review]
LGTM !

**Pull request merged**